nnstreamer/tensor_filter/tensor_filter_nntrainer.hh

```diff
@@ -113,12 +109,11 @@ private:
   /// model->compile                     (available)
   /// model->initialize                  (available)
   /// model->readModel                   (available)
-  /// model->inference                   (n/a)
+  /// model->inference                   (available)
   /// model->getInputDimension           (n/a)
```

Comment:
once this is available with #1430, I will update the implementation to depend only on cc-API.

Reply:
nice